force_wfmanager/wfmanager_global_task.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import platform
+
+from pyface.tasks.action.api import SMenu, SToolBar, TaskAction
+from pyface.api import ImageResource
+from traits.api import List
+
+from force_wfmanager.wfmanager import TaskToggleGroupAccelerator
+
+from traits.api import Callable
+from envisage.ui.tasks.task_extension import TaskExtension
+from pyface.tasks.action.api import SchemaAddition
+
+log = logging.getLogger(__name__)
+
+IS_WINDOWS = (platform.system() == "Windows")
+
+
+class ExitAction(TaskAction):
+    """ A standard 'Quit' (Mac) or 'Exit' (Windows) menu action. """
+
+    accelerator = "Alt+F4" if IS_WINDOWS else "Ctrl+Q"
+    method = "setup_task.exit"
```

Comment:
I think Corran's suggestion meant to move this method reference to the `TaskApplication` object (`WfManager`, rather than `WfManagerSetupTask`).

This does not seem straight forward, so it is something probably best dealt with in an additional PR